com.unity.render-pipelines.universal/Samples~/RenderPassSamples/BlurryRefraction/RunPassAtRuntime.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public class RunPassAtRuntime : MonoBehaviour
+{
+    public KawaseBlurSettings setting;
+    KawaseBlur renderPass;
+
+    bool useRenderPass = false;
+
+    Material refactiveGlass;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        refactiveGlass = GetComponent<Renderer>().material;
+        renderPass = new KawaseBlur("KawaseBlur", setting);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        renderPass.renderPassEvent = setting.renderPassEvent;
+        if(Input.GetKeyDown(KeyCode.C))
+        {
+            useRenderPass = !useRenderPass;
+            refactiveGlass.SetFloat("IsUsingRenderFeature", useRenderPass ? 1.0f : 0.0f);
+
+        }
+        if(useRenderPass)
+        {
+            ((UniversalRenderPipelineAsset)GraphicsSettings.currentRenderPipeline).scriptableRenderer.EnqueuePass(renderPass);
```

Comment:
The camera can override the renderer. So we need to get the renderer from the camera. 

There are multiple ways to do it:
1) you can get a reference to the camera. Then call `camera.GetUniversalAdditionalCameraData().scriptableRenderer`
2) you can use beginCameraRendering event https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@10.2/manual/using-begincamerarendering.html to get the camera

To make it easier to users we could actually even add a GetRenderer() from the camera
so `camera.GetUniversalAdditionalCameraData().scriptableRenderer` could be just camera.GetScriptableRenderer()`

Reply:
Should I create the shortcut in this PR?

Reply:
This is included in the next commit
But docs still missing for it